data/remnant missions.txt

```diff
@@ -139,6 +139,99 @@ mission "Remnant Blood Test Patch"
+			choice
+				`	"Thanks for the help. I'll go fit this on my ship."`
+				`	"How'd you get to Republic space?"`
+					goto republic
+			
+			`	You say goodbye to the woman and begin walking to your ship. "Safe travels, Captain," she sings with a big grin on her face as you walk away.`
+				decline
+			
+			label republic
+			`	"I was an adventurous spirit in my youth," she says. "One day I found my way out of the Ember Waste and back to what I learned was the rest of humanity. For years my people had hidden here believing the Alphas had killed everyone else, but I discovered that we had little to fear.`
+			`	"I explored the Republic for many months, learning what had happened since the Alpha Wars, but when I attempted to return to the Ember Waste to tell my people, I found that my key stone had been stolen at some point during my journey. I had no way to return."`
+			`	She lets out a long sigh. "I don't want to bore you with all the details, but I resigned myself to my new life. I met a man, had two daughters, but never lost hope of returning to my true home. One day, after my daughters had grown and my husband had passed in an accident, I decided to go on another adventure. I dusted off my old ship, and soon discovered Hai space."`
+			branch hai
+				has "First Contact: Hai"
+			choice
+				`	"The who?"`
+			`	"How you've become lost here without having discovered the Hai first is beyond me." She chuckles. "The Hai are a peaceful alien people north of humanity. Go looking for a purple wormhole in the uninhabited systems of the Far North. You may find your way there.`
+				goto next
+			
+			label hai
+			choice
+				`	"You've met the Hai?"`
+			`	"Well of course. How else would I have been able to return home?`
+			
+			label next
+			`	"To my surprise, the Hai had a large number of quantum key stones." She pauses for a moment. "Might be a good idea for you to bring some of those Hai key stones here. Anyways, I was so thrilled that I didn't even ask any questions about how they had them. I bought one and flew as fast as I could back to the Ember Waste, and I've been here ever since."`
+			choice
+				`	"How long ago was that?"`
+				`	"What about your daughters?"`
+					goto daughters
+			
+			`	"Well I got lost when I was 17 and returned when I was 64, so perhaps some 130 years ago now that I returned." She chuckles at the surprised look on your face as you realize that she is nearly 200 years old. "Uncovering ancient alien technology is quite the blessing.`
+				goto end
+			
+			label daughters
+			`	"Oh, I made sure to tell them I was returning home. They know where to find me if they ever need me, and I have made trips to see them every decade or so. I'm nearing the end of my days, though, so I may only have one more trip left in me.`
+			
```

Comment:
Given how secretive the Remnant are, I don't think it is in character for them to casually let strangers (which her daughters would be) know where to find her. Might be better to change the sentence about "They know where to find me..." to something along the lines of "They think I am wandering around exploring the galaxy, and I left a QEC with them so they can send me messages when they need to."

For one, this change fits better with the Remnant culture of secrecy/security. For another, this opens up a potential venue for future missions, as the lady could come find the player to get help dealing with some problem that her daughters have let her know about.

Reply:
>Given how secretive the Remnant are, I don't think it is in character for them to casually let strangers (which her daughters would be) know where to find her.

My intention with that was to show how she's rather different from the rest of the Remnant (not as secretive, didn't fear leaving the Ember Waste, doesn't sing (although that's just her not singing around you since you're from the Republic)), but I'm alright with changing this one part.

Reply:
Maybe just change the wording a bit to imply it hasn't been very often. I think it might negatively impact the implications of the Remnant's isolation and lack of contact with humanity if they already have someone going back and forth regularly. But I don't see a problem with mentioning that she's gone back and forth a couple times. Especially if the way she says it implies that it is a long and/or difficult and/or dangerous trip.

Reply:
Something I just realized too; the assumption here is that her daughters are still alive, which means that her daughters are at least say 150 years old.
As far as I'm aware, there's zero mention in game of the typical age that people live to in Republic space. But, this mission does make it seem that living for 200 years is remarkable and out of the ordinary, so it could be that no one lives near that long in the Republic. 

Something I mentioned on Discord is that while IRL life expectancy has increased with time, the ages of the oldest living people have not seemed to increase in the same way; we get more old people, but not people living older than anyone prior, that being a max of about 120 years. So, I think it'd make sense to say that her daughters passed decades ago, because "It seems like the Republic can't quite get anyone to live past 130 years," or something like that.

Reply:
That makes sense, and would work for this.

Reply:
You could go for comments about visiting grand or great-grand kids instead.